dkg/runner.go

```diff
@@ -69,78 +76,79 @@ func (r *Runner) ProcessMsg(msg *SignedMessage) (bool, map[types.OperatorID]*Sig
 				Signature: sig.Serialize(),
 			}
 			if err := r.signAndBroadcastMsg(pdd, DepositDataMsgType); err != nil {
-				return false, nil, errors.Wrap(err, "could not broadcast partial deposit data")
+				return false, errors.Wrap(err, "could not broadcast partial deposit data")
 			}
 			r.DepositDataSignatures[r.Operator.OperatorID] = pdd
 		}
-		return false, nil, nil
+		return false, nil
 	case DepositDataMsgType:
 		depSig := &PartialDepositData{}
 		if err := depSig.Decode(msg.Message.Data); err != nil {
-			return false, nil, errors.Wrap(err, "could not decode PartialDepositData")
+			return false, errors.Wrap(err, "could not decode PartialDepositData")
 		}
 
 		if err := r.validateDepositDataSig(depSig); err != nil {
-			return false, nil, errors.Wrap(err, "PartialDepositData invalid")
+			return false, errors.Wrap(err, "PartialDepositData invalid")
 		}
 
 		if found := r.DepositDataSignatures[msg.Signer]; found == nil {
 			r.DepositDataSignatures[msg.Signer] = depSig
 		} else if !bytes.Equal(found.Signature, msg.Signature) {
-			return false, nil, errors.New("inconsistent partial signature received")
+			return false, errors.New("inconsistent partial signature received")
 		}
 
 		if len(r.DepositDataSignatures) == int(r.InitMsg.Threshold) {
 			// reconstruct deposit data sig
 			depositSig, err := r.reconstructDepositDataSignature()
 			if err != nil {
-				return false, nil, errors.Wrap(err, "could not reconstruct deposit data sig")
+				return false, errors.Wrap(err, "could not reconstruct deposit data sig")
 			}
 
 			// encrypt Operator's share
-			encryptedShare, err := r.config.Signer.Encrypt(r.Operator.EncryptionPubKey, r.KeyGenOutput.Share.Serialize())
+			encryptedShare, err := r.config.Signer.Encrypt(r.Operator.EncryptionPubKey, r.KeygenOutcome.KeyGenOutput.Share.Serialize())
```

Comment:
encrypt receives the RSA pub key not the share

Reply:
The first argument is the rsa pub key and second is the plaintext (share) to be encrypted.